frontend/src/app/event/event-filter/event-filter.pipe.ts

```diff
@@ -24,6 +24,11 @@ export class EventFilterPipe implements PipeTransform {
       return a.time.getTime() - b.time.getTime();
     });
 
+    // Filter out past dates
+    events = events.filter((e: Event) => {
+      return e.time.getTime() > new Date().getTime()
```

Comment:
We should still be doing pagination at the API level for viewing past events and only typically loading future events.